packages/react-native-tracer-provider/native-src/android/app/dependencies.gradle

```diff
@@ -0,0 +1,4 @@
+dependencies {
+  api "io.embrace:embrace-android-sdk:6.13.0"
```

Comment:
do we want to update this version manually here in the future?

Reply:
this ends up getting rewritten by the `update_android_version.ts` script